### DISCUSSION TITLE
Incoming email webhooks?

### DISCUSSION THREAD

**Discussion by rodel1bit** [Mailu/Mailu#3196] [User support] [answered]
Hi is there any chance that having this feature?

**Comment by nextgens** [comment]
You already have multiple ways of achieving that with the current codebase.

1) using sieve scripts, see https://doc.dovecot.org/configuration_manual/sieve/plugins/extprograms/#full-examples
2) using rspamd, see https://rspamd.com/doc/modules/metadata_exporter.html 

all of it can be configured with overrides; it just needs to be documented.

**Reply by rodel1bit** [reply]
Thank you

**Reply by rodel1bit** [reply]
Where do i find the directory of sieve?

**Reply by nextgens** [reply, marked as answer]
Somewhere in ``/usr/lib/dovecot/`` on the imap container

**Reply by rodel1bit** [reply]
Thank you

**Reply by rodel1bit** [reply]
How about the rspamd?
